app/views/errors/unprocessable_entity.html.erb

```diff
@@ -1,15 +1,11 @@
 <%= content_for :page_title, 'Sorry, there’s a problem with the service' %>
 
-<div class="nhsuk-grid-row">
-  <div class="nhsuk-grid-column-two-thirds">
-    <h1 class="nhsuk-heading-l">Sorry, there’s a problem with the service</h1>
```

Comment:
I think removing two-thirds from here means that the regular error pages at /422 and /404 don't render correctly. We probably need to add the `layout :two_thirds` to the `ErrorsController`.